ci/dra.sh

```diff
@@ -11,14 +11,20 @@ if [ -z "$VERSION_QUALIFIER_OPT" ]; then
 else
   VERSION_QUALIFIER="$VERSION_QUALIFIER_OPT" RELEASE=1 rake artifact:all
 fi
-echo "GENERATED ARTIFACTS"
-for file in build/logstash-*; do shasum $file;done
 
 STACK_VERSION=`cat versions.yml | sed -n 's/^logstash\:\s\([[:digit:]]*\.[[:digit:]]*\.[[:digit:]]*\)$/\1/p'`
 
+echo "Saving tar.gz for docker images"
+docker save docker.elastic.co/logstash/logstash:${STACK_VERSION}-SNAPSHOT | gzip -c > build/logstash-${STACK_VERSION}-docker-image-x86_64.tar.gz
+docker save docker.elastic.co/logstash/logstash-oss:${STACK_VERSION}-SNAPSHOT | gzip -c > build/logstash-oss-${STACK_VERSION}-docker-image-x86_64.tar.gz
+docker save docker.elastic.co/logstash/logstash-ubi8:${STACK_VERSION}-SNAPSHOT | gzip -c > build/logstash-ubi8-${STACK_VERSION}-docker-image-x86_64.tar.gz
+
+echo "GENERATED ARTIFACTS"
+for file in build/logstash-*; do shasum $file;done
+
 echo "Creating dependencies report for ${STACK_VERSION}"
-mkdir -p build/reports/dependencies-reports/
-bin/dependencies-report --csv=build/reports/dependencies-reports/logstash-${STACK_VERSION}.csv
+mkdir -p build/distributions/dependencies-reports/
```

Comment:
The PR gives no information about the reason for this change, can you clarify please?

Reply:
You are right, this change is required by the `release-manager` that needs the reports to be present in the folder `build/distributions/dependencies-reports/` instead of `build/reports/dependencies-reports/`.
However we could keep the original and put the file in the expected folder in a follow up PR when the artifacts are placed in the right position to be uploaded.
WDYT?

Reply:
It's ok to keep both in these changes in the PR, just include the reasons in the commit.

Reply:
ok, done in 3d9041cb29da67bf6b8110a03886644e6e059804